apps/docs/content/components/Section/react.mdx

```diff
@@ -0,0 +1,151 @@
+---
```

Comment:
I don't know why this is, but when I view the docs on the preview link, when I'm on the _React_ page I no longer see the navigation to go back to _Overview_.

![image](https://github.com/user-attachments/assets/44ad6c7f-425a-4678-a64b-efca6d4e04ca)

^^ This disappears on https://primer-9706701c7a-26139705.drafts.github.io/brand/components/Section/react